src/stores/NotificationStore.ts

```diff
@@ -0,0 +1,51 @@
+import { Slide, SnackbarProps, PaperProps } from '@mui/material'
+import { NotificationLayoutData } from 'components/layout/NotificationSnackBar/DonationNotificationLayout'
+import { makeAutoObservable } from 'mobx'
+import { enableStaticRendering } from 'mobx-react'
```

Comment:
I personally think that we should stray away from `mobx` and class based implementations and go for something hook based, considering the whole react ecosystem seems to be moving to hooks. Something like [zustand](https://docs.pmnd.rs/zustand/getting-started/introduction) I think would be more fitting for our stores, which is way more lightweight than `mobx` too 

Let's see the opinion of other people though.

Reply:
I don't see a problem with mobx using classes and react using hooks. If zustand will improve the performance of the app ok =]

Reply:
> I don't see a problem with mobx using classes and react using hooks. If zustand will improve the performance of the app ok =]

Technically speaking there isn't a problem. I just don't like having both paradigms when the other is an option and the concept of the passing around the `this` and general observable magic that comes with it.

In terms performance and size of `mobx` + `mobx-react` 16kb + 5.5kb and `zustand` is 1.1kb, but we are talking in kb here so it's not a huge deal

Reply:
These technologies were picked from someone before us to serve a purpose and it wasn't before React introduced Hooks. I personally don't mind mixing up paradigms. About mobx it has some powerful features and it's easy to use.

Reply:
I agree we should depend on least possible libs and in this case it doesn't seem we need the mobx even that it already existed in the project.
See this article - the guy is presenting a working solution without additional need of mobx or any additional state management: https://blog.logrocket.com/implementing-websocket-communication-next-js/

Reply:
Sure I will remove mobx from the socket implementation